### PR TITLE
Add new section for test integrations

### PR DESCRIPTION
**Pull request by LeoColman**
This commit adds documentation on how Testcontainers can be integrated with the Kotest framework.

From the very old #2533 it was decided that a reference to Kotest would be great in the docs. This commit adds it.

### CODE REVIEW

File: docs/test_framework_integration/kotest.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Kotest
+
+[Kotest](https://github.com/kotest/kotest) extension for [Testcontainers](https://github.com/testcontainers/testcontainers-java) library, which allows to use Docker containers inside of Kotest tests.
+
+The source code is maintained at [Kotest Extensions TestContainers](https://github.com/kotest/kotest-extensions-testcontainers)
```
Comment (eddumelendez) [inline docs/test_framework_integration/kotest.md:5]:
```suggestion
The source code is maintained at [Kotest Extensions Testcontainers](https://github.com/kotest/kotest-extensions-testcontainers)
```

File: docs/test_framework_integration/kotest.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Issues and requests are maintained at the [Kotest Repository](https://github.com/kotest/kotest)
+
+## Documentation
+
+The full and updated documentation for Kotest integration with TestContainers can be found at [kotest.io](https://kotest.io/docs/extensions/test_containers.html)
```
Comment (eddumelendez) [inline docs/test_framework_integration/kotest.md:10]:
```suggestion
The full and updated documentation for Kotest integration with Testcontainers can be found at [kotest.io](https://kotest.io/docs/extensions/test_containers.html)
```

**Review by eddumelendez — CHANGES_REQUESTED** [review]
Thanks for the contribution! I've left some comments

**Comment by LeoColman** [comment]
Done :)

**Comment by eddumelendez** [comment]
Hi @LeoColman, thinking more about this... I think we should not duplicate the existing documentation in kotest but we should point to it. what do you think about renaming the file to integration and then add Kotest with a link to https://kotest.io/docs/extensions/test_containers.html

The content can says something like `The following open source projects use Testcontainers`

**Comment by LeoColman** [comment]
I agree!

I even tried to make it smaller to avoid maintenance issues.

How could I apply your suggestion?

Apr 20, 2023 19:46:14 Eddú Meléndez Gonzales ***@***.***>:

**Comment by eddumelendez** [comment]
Just update this PR :)

**Comment by LeoColman** [comment]
Alright, I renamed it to `external` and replaced the documentation with links to the module itself.


Also took the opportunity to add another integration from #3298 to the same page. <-- @jlink @gtrefs

**Comment by LeoColman** [comment]
I thought of adding a `last updated` or `last maintained` to each of them, to prevent accidentally using a very old framework. Let me know what you think @eddumelendez

File: docs/test_framework_integration/external.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# External Integrations
+
+The following Open Source frameworks add direct integration to Testcontainers
+
+
```
Comment (eddumelendez) [inline docs/test_framework_integration/external.md:5]:
```suggestion
```

**Review by eddumelendez — APPROVED** [review]

**Comment by eddumelendez** [comment]
Thanks for your contribution, @LeoColman ! This will be available soon!

**Pull request merged**